### ISSUE TITLE
Redux Dev Tools Extension version 2.16.0 clashing my working code

### ISSUE THREAD

**Issue by benson7667** [zalmoxisus/redux-devtools-extension#590]
My code is working fine when the extension installed is **version 2.15.3** or i removed the redux dev tools extension.

The error msg as below:
**_Error: You may not call store.getState() while the reducer is executing. The reducer has already received the state as an argument. Pass it down from the top reducer instead of reading it from the store._**

Here is my store.js:

```
import { createStore, applyMiddleware, compose } from 'redux';
import thunk from 'redux-thunk';
import rootReducer from './reducers';

const initialState = {};
const middleware = [thunk];

/* eslint-disable no-underscore-dangle */
const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
/* eslint-enable */

const store = createStore(
  rootReducer,
  initialState,
  composeEnhancers(applyMiddleware(...middleware))
);

export default store;

```

**Comment by danivivriti** [comment]
Getting same error, exact same configs of @jithinkmatthew

**Comment by felthy** [comment]
I've also been investigating this. In my case at least, the error only seems to occur when dispatching an action immediately after creating the store. If I don't do that, actions dispatched later seem to be ok. I've narrowed the cause down to this line:

https://github.com/zalmoxisus/redux-devtools-extension/blob/6578cdbb3b134e5e57f27cf9544e45c7576a8b4b/src/browser/extension/inject/pageScript.js#L247

which was altered 9 hours ago in commit 5f94fc09. So this is only triggered when `getMaxAge()` is called with a `liftedAction` param.

**Comment by mattdevio** [comment]
Same as #588

**Comment by felthy** [comment]
Ah yes https://github.com/zalmoxisus/redux-devtools-extension/issues/588#issuecomment-442319942 beat me to it!

**Comment by mattdevio** [comment]
Has more information, but that issue is really noisy with the +1s all over the place.

**Comment by felthy** [comment]
And even #588 was a dupe of #587

**Comment by GabrielFerrarini** [comment]
Changing webpack mode from development to productions makes the issue disappear.

**Comment by nuclearspike** [comment]
Recommended simplest action: Disable extension and subscribe to this issue and issue 588 and 587 to know when you can re-enable.

Otherwise, follow instructions in issue 588 to disable 2.16 and manually download the 2.15.5 zip extension and manually add it to Chrome if you just can't wait. https://github.com/zalmoxisus/redux-devtools-extension/issues/588#issuecomment-442332672

**Comment by saadbinsaeed** [comment]
Revert back to the previous version v2.15.5 

- Download [redux-devtool-extension](https://github.com/zalmoxisus/redux-devtools-extension/releases/download/2.15.5/extension.zip)
- Extract the zip
- Type `chrome://extensions` into the url
- toggle on developer mode on the top right of the page.
- Click on the **Load Unpacked** button. After clicking the button, Select the extracted folder.

Hope it will help :)

**Comment by nitimmerman** [comment]
hmm, still the same issue for me: 
```
core.js:1673 ERROR TypeError: Cannot read property 'whitelist' of undefined
    at isFiltered (<anonymous>:1:5016)
    at Object.x [as send] (<anonymous>:1:73900)
    at DevtoolsExtension.notify (extension.ts:111)
    at ScanSubscriber.StoreDevtools.liftedAction$.pipe.Object.state [as accumulator] (devtools.ts:82)
    at ScanSubscriber._tryNext (scan.js:61)
    at ScanSubscriber._next (scan.js:54)
    at ScanSubscriber.Subscriber.next (Subscriber.js:54)
    at WithLatestFromSubscriber._next (withLatestFrom.js:66)
```

**Comment by manklu** [comment]
@nitimmerman That's https://github.com/zalmoxisus/redux-devtools-extension/issues/589

**Comment by 951565664** [comment]
watch

**Comment by Krzychek** [comment]
> watch
There's subscribe button for this on the side panel. And it doesn't send notifications to everyone.

**Comment by 951565664** [comment]
> > watch
> 
> There's subscribe button for this on the side panel. And it doesn't send notifications to everyone.

Sorry, I left a comment that was useless, I have deleted it.

**Comment by zalmoxisus** [comment]
@nitimmerman that's fixed in last version, refer to #589.

**Comment by nitimmerman** [comment]
thanks! I've already commented in #589 as well
